### PR TITLE
Implement generating HTML for a formula with objects

### PR DESCRIPTION
**Pull request by cwayfinder**

### CODE REVIEW

**Review by LimaBD — APPROVED** [review]
Tests cases for interpreteing nested object passing

File: excel.test.ts [hunk 1]
```diff
@@ -96,10 +96,43 @@ describe('Excel.toHtml() end usage tests', () => {
...
+                        method: <span class="value">'GET'</span>, 
+                        url: <span class="value">'https://api.github.com/users/defunkt'</span>
+                    }
+                    <span class="paren-deep-1">)</span>
+                </div>
+            `));
+    });
+
+    test('Testing parsing nested objects', () => {
+        expect(excel.toHtml(`HTTP({method: 'GET', url: 'https://api.github.com/users/defunkt', headers: { 'User-Agent': 'request' }})`))
```
Comment (cwayfinder) [inline excel.test.ts]: Here 'User-Agent' is quoted

File: excel.test.ts [hunk 2]
```diff
@@ -96,10 +96,43 @@ describe('Excel.toHtml() end usage tests', () => {
...
+          .toEqual(removeExtraSpaces(`
+                <div>
+                    <span class="function">HTTP</span>
+                    <span class="paren-deep-1">(</span>
+                    {
+                        method: <span class="value">'GET'</span>, 
+                        url: <span class="value">'https://api.github.com/users/defunkt'</span>,
+                        headers: {
+                            User-Agent: <span class="value">'request'</span>
```
Comment (cwayfinder) [inline excel.test.ts:124]: And here User-Agent is not quoted. Can we make it consistent?

Comment (cwayfinder) [reply excel.test.ts:124]: In JS, object keys with hyphens should be in quotes. But here in a formula, we probably can go without quotes.

Comment (LimaBD) [reply excel.test.ts:124]: fixed!, now any combination of these:
```
excel.toHtml(`HTTP( { method: 'GET' } )
excel.toHtml(`HTTP( { 'method': 'GET' } )
excel.toHtml(`HTTP( { "method": 'GET' } )
```
Produce the same HTML output.

**Pull request merged**